api/src/main/java/org/hawkular/alerts/actions/api/ActionMessage.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.actions.api;
```

Comment:
Shouldn't we change those?

Reply:
I didn't change any old code packages yet. Does it matter?

Reply:
Not sure if it matters to do it now or later, but we have at least agreed to use `com.redhat.cloud.custompolicies.engine` as base package here.

Reply:
To me it would make sense that as we refactor these to use "Quarkus" for injections and that sort of updates we would move the packages. It would make it easy to track when the app is still using non-refactored code.